### PR TITLE
security: add private distributor list, and its application template

### PR DESCRIPTION
**Pull request by gyuho**
Signed-off-by: Gyuho Lee <leegyuho@amazon.com>

Please read https://github.com/etcd-io/etcd/blob/master/CONTRIBUTING.md#contribution-flow.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/distributors-application.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+name: Distributors Application
+title: Distributors Application for <YOUR DISTRIBUTION HERE>
+about: Apply for memborship of security@etcd.io
```
Comment (hexfusion) [inline .github/ISSUE_TEMPLATE/distributors-application.md:4]: nit: s/memborship/membership

**Review by hexfusion — COMMENTED** [review]
LGTM thanks @gyuho

**Review by spzala — COMMENTED** [review]
lgtm.
Thanks @gyuho

**Comment by wpedrak** [comment]
Hi @gyuho, I've just submitted https://github.com/etcd-io/etcd/pull/12795  and it seems that this (#12798) PR is breaking one of our tests. 
```
'markdown_you' started at Wed Mar 24 08:34:39 UTC 2021
% 'eval' 'find . -name \*.md ! -path '\''*/vendor/*'\'' ! -path '\''./Documentation/*'\'' ! -path '\''./gopath.proto/*'\'' ! -path '\''./release/*'\'' -exec grep -E --color '\''[Yy]ou[r]?[ '\''\'\'''\''.,;]'\'' {} + || true'
FAIL: 'markdown_you_find_eschew_you' checking failed (printed output)
./.github/ISSUE_TEMPLATE/distributors-application.md:**1. Have a user base not limited to your own organization.**
./.github/ISSUE_TEMPLATE/distributors-application.md:**7. Have someone already on the list vouch for the person requesting membership on behalf of your distribution.**
./security/README.md:This list provides actionable information regarding etcd security to multiple distributors. Members of the list may not use the information for anything other than fixing the issue for respective distribution's users. If you continue to leak information and break the policy outlined here, you will be removed from the list.
FAIL: 'markdown_you' failed at Wed Mar 24 08:34:39 UTC 2021
```
Complete build log is here: https://travis-ci.com/github/etcd-io/etcd/jobs/493314455
Could you address it?

**Comment by ptabor** [comment]
@nate-double-u 

Now when nearly all documentation is in external repository, I doubt its worth to maintain this "test": 

```
function markdown_you_find_eschew_you {
  local find_you_cmd="find . -name \\.md ! -path '/vendor/' ! -path './Documentation/' ! -path './gopath.proto/' ! -path './release/' -exec grep -E --color '[Yy]ou[r]?[ '\\''.,;]' {} + || true"
```

That affects md files but not in 'Documentation'. 

We should either: 
  - get rid of it completely
  - or tune it to catch more specific patterns ([Yy]ou should, You have to, You must) and apply to `*.go` comments as well.

**Pull request merged**